husky_directory/templates/full_results.html

```diff
@@ -14,7 +14,14 @@ <h4>{{scenario['description']}}</h4>
                     <h3>{{ data['name'] }}</h3>
                     <ul class="dir-listing">
                         <li>{{ data['email'] }}</li>
-                        <li>{{ data['phone'] }}</li>
```

Comment:
RE: change is displaying the phone(s)
This same kind of change is needed `summar_results.html`